Makefile

```diff
@@ -8,7 +8,7 @@ WARN_COLOR=\33[33m
 
 .PHONY: all start build test test-integration coverage deps help fmt vet lint tools
 
-VERSION=`cat .version`
+VERSION=`git describe --tag --always`
```

Comment:
VERSION=`make bump-version` (Inseption hehhe)

I am getting the wrong version with this `git describe --tag --always` the way we do today is `git tag --sort=committerdate | tail -n 1`

Reply:
Sorry, I misunderstood your primary idea.  The `git describe --tag --always` looks great. 
Is it updated for each tag also?